src/main/java/com/epam/restbookservice/controllers/AccountController.java

```diff
@@ -0,0 +1,58 @@
+package com.epam.restbookservice.controllers;
+
+import com.epam.restbookservice.domain.Role;
+import com.epam.restbookservice.domain.User;
+import com.epam.restbookservice.dtos.UserDTO;
+import com.epam.restbookservice.services.UserService;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.List;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("/account")
```

Comment:
```suggestion
@RequestMapping("/accounts")
```

Reply:
We can use this class only to manage current account and not multiple. So there's no need in this change